sdks/python/apache_beam/transforms/core.py

```diff
@@ -1633,7 +1636,19 @@ def with_exception_handling(
         use_subprocess,
         threshold,
         threshold_windowing,
-        timeout)
+        timeout,
+        error_handler)
+
+  def with_error_handler(self, error_handler, **exception_handling_kwargs):
```

Comment:
Could we also have a unit test that would show how the `**exception_handling_kwargs` can be used?

Reply:
Good call. Done.